packages/card/src/vaadin-card.js

```diff
@@ -306,6 +343,18 @@ class Card extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
     `;
   }
 
+  get titleHeadingLevel() {
+    return this.__titleHeadingLevel;
+  }
+
+  set titleHeadingLevel(value) {
+    if (value == null) {
+      this.__titleHeadingLevel = null;
+    } else if (value >= 1 && value <= 6) {
+      this.__titleHeadingLevel = value;
+    }
+  }
```

Comment:
If the point of this logic is to ignore invalid values, it's better to use `_titleHeadingLevelChanged` observer and have `console.warn()` there with setting value to `null`. See for example `vaadin-integer-field.js`

Reply:
Before making the change, what do you think about limiting the value? I have seen comments in online discussions that this is also used to go beyond 6 and help developers have more heading layers.

Reply:
Removed validation and renamed the property to `titleHeadingLevel`.